### DISCUSSION TITLE
2 Types of users to signup...

### DISCUSSION THREAD

**Discussion by thecybergypsy** [supabase/supabase#14693] [Questions]
I am nicely adding a new user to public.profile  when they have signed up, but I want to create two types of user on two separate tables, for example public.profile and public.profile_2 with different data.  Triggering a function isn't any use because this will only put the data onto a single table.

**Comment by GaryAustin1** [comment]
You should provide more info.  Not really clear what you are seeing as we don't know your operations, tables, errors, where that "flag" is, etc.
